maps/expert/dark_castle/chimps.ahk

```diff
@@ -0,0 +1,135 @@
+DarkCastleChimps() {
+    global TS := Map(
+        "Obyn",["hero",[865,447]],
+        "Dart A",["dart",[739,656]],
+        "Dart B",["dart",[688,691]],
+        "Sub A",["sub",[1083,690]],
+        "Sub B",["sub",[1086,398]],
+        "Alch A",["alch",[1011,657]],
+        "Alch B",["alch",[666,388]],
+        "Alch C",["alch",[711,346]],
+        "Druid A",["druid",[715,433]],
+        "Druid B",["druid",[789,447]],
+        "Druid C",["druid",[941,448]],
+        "Druid D",["druid",[766,384]],
+        "Druid E",["druid",[842,384]],
+        "Druid F",["druid",[918,385]],
+        "Spike",["spike",[1022,441]],
+        "Village",["village",[869,303]]
+    )
+
+    Place("Dart A")
+    Place("Sub A")
+
+    StartGame()                     ; Round 6

```

Comment:
Round numbers are no longer included as comments but this reminder of the first round may stay if you want it to.

Reply:
removed